docs/best-practices/continuous-integration.md

```diff
@@ -141,25 +142,77 @@ and run
 gem install bundler && bundle update
 ```
 
-This will create a `Gemfile.lock`, that defines all Ruby dependencies. Make sure to commit both files to version control.
+This will create a `Gemfile.lock` that defines all Ruby dependencies.  Make sure
+to commit both files to version control.
 
-Next, use the following `circle.yml` file
+Next, add the following to your `Fastfile`:
+
+```ruby
+# fastlane/Fastfile
+
+...
+platform :ios do
+  before_all do
+    setup_circle_ci
+  end
+  ...
+end
+```
+
+The `setup_circle_ci` _fastlane_ action will perform the following actions:
+
+* Create a new temporary keychain for use with
+  [match](https://fastlane.tools/match) (see the [CircleCI code signing
+  doc](https://circleci.com/docs/2.0/ios-codesigning/) for more details).
+* Switch _match_ to readonly mode to make sure CI does not create new
+  code signing certificates or provisioning profiles.
+* Set up log and test result paths to be easily collectible.
+
+Next, create a `.circleci` directory in your project and add a
+`.circleci/config.yml` with the following content:
 
 ```yml
-machine:
-  xcode:
-    version: "7.3"
-dependencies:
-  override:
-    - bundle check --path=vendor/bundle || bundle install --path=vendor/bundle --jobs=4 --retry=3 --without development
-  cache_directories:
-    - vendor/bundle
-test:
-  override:
-    - bundle exec fastlane test
+# .circleci/config.yml
+
+version: 2
+jobs:
+  build:
+    macos:
+      xcode: "9.0"
+    working_directory: /Users/distiller/output
+    environment:
+      FL_OUTPUT_DIR: $CIRCLE_WORKING_DIRECTORY
+      FASTLANE_LANE: test
+    shell: /bin/bash --login -o pipefail
+    steps:
+      - checkout
+      - restore_cache:
+          key: 1-gems-{{ checksum "Gemfile.lock" }}
+      - run: bundle check || bundle install --path vendor/bundle
+      - save_cache:
+          key: 1-gems-{{ checksum "Gemfile.lock" }}
+          paths:
+            - vendor/bundle
+      - run:
+          name: fastlane
+          command: bundle exec fastlane $FASTLANE_LANE
+      - run:
+          command: cp $FL_OUTPUT_DIR/scan/report.junit $FL_OUTPUT_DIR/scan/results.xml
+          when: always
+      - store_artifacts:
+          path: /Users/distiller/output
```

Comment:
I think this will store your entire project in artifacts.

I think we want:
`working_directory: $HOME/project`
`FL_OUTPUT_DIR: $CIRCLE_WORKING_DIRECTORY/output`
`store_*: path: /Users/distiller/project/output`